Update  Sylius 1.12 with PriceHistoryPlugin to Sylius 1.13 upgrade file (#15084)
Sylius/Sylius: UPGRADE-FROM-1.12-WITH-PRICE-HISTORY-PLUGIN-TO-1.13-LEGACY.md

    ```

> 1. The class `Sylius\PriceHistoryPlugin\Infrastructure\EventSubscriber\ChannelPricingLogEntryEventSubscriber` has been renamed to `Sylius\Bundle\CoreBundle\PriceHistory\EventListener\ChannelPricingLogEntryEventListener`.
>    Additionally, it no longer implements `Doctrine\Bundle\DoctrineBundle\EventSubscriber\EventSubscriberInterface` as it has been deprecated.
> 
>    If you have used this class in your project, you should make the necessary adjustments.
> 

Reviewer: ```suggestion
1. The class `Sylius\PriceHistoryPlugin\Infrastructure\EventSubscriber\ChannelPricingLogEntryEventSubscriber` has been replaced by `Sylius\Bundle\CoreBundle\PriceHistory\EventListener\ChannelPricingLogEntryEventListener`.
```

In my opinion that's enough.
